packages/common/src/api/tan-query/useUsers.ts

```diff
@@ -4,21 +4,17 @@ import { useDispatch } from 'react-redux'
 import { userMetadataListFromSDK } from '~/adapters/user'
 import { useAudiusQueryContext } from '~/audius-query'
 import { Id, ID } from '~/models/Identifiers'
-import { Kind } from '~/models/Kind'
-import { User } from '~/models/User'
-import { addEntries } from '~/store/cache/actions'
-import { EntriesByKind } from '~/store/cache/types'
 import { removeNullable } from '~/utils/typeUtils'
 
 import { QUERY_KEYS } from './queryKeys'
 import { Config } from './types'
 import { primeUserData } from './utils/primeUserData'
 
-export const useUsers = (userIds: ID[], config?: Config) => {
+export const useUsers = (userIds?: ID[], config?: Config) => {
```

Comment:
for arguments that you ought to pass but are optional in case the input is undefined, I prefer `userIds: ID[] | null | undefined` since it'll prompt at the call site if no positional arg is passed

Since `useUsers()` isn't a valid use case, I think that is desirable

Reply:
Yeah great idea